src/components/app/ProfileSharing.js

```diff
@@ -76,22 +76,26 @@ type ProfileSharingCompositeButtonProps = {
   onProfilePublished: typeof actions.profilePublished,
 };
 
-class ProfileSharingCompositeButton extends PureComponent {
+type ProfileSharingCompositeButtonState = {
+  state: string,
+  uploadProgress: number,
+  hash: string,
+  error: Error | null,
+  fullUrl: string,
+  shortUrl: string,
+};
+
+class ProfileSharingCompositeButton extends PureComponent<
+  ProfileSharingCompositeButtonProps,
+  ProfileSharingCompositeButtonState
+> {
   props: ProfileSharingCompositeButtonProps;
```

Comment:
I believe this `props: Props` is a duplication of the `PureComponent<Props, State>` statement.

Reply:
ooops, that's actually a left over, thanks !